### PR TITLE
Abuse indicators

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
Ready

## Related Issues
related to: https://github.com/demisto/etc/issues/17949

## Description
Fixed Indicators to match the new format
## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
AbuseDB-Packaging|https://github.com/demisto/content/pull/4228

## Does it break backward compatibility?
   - No

- [x] [Docuentation](https://github.com/demisto/etc/issues/19376)

### CODE REVIEW

**Comment by orlichter1** [comment]
@yaakovi in the packaging PR I've had to change some things using the plugin. I believe there it'll change to this format so after I'll merge the packaging PR I'll see if there's a need to change. In general, maybe we should run this sorted thing the plugin is using when running the packaging script.

File: Integrations/AbuseDB/AbuseDB.yml [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+    name: abuseipdb-get-categories
+    outputs:
+    - contextPath: AbuseIPDB.Categories
+      description: List of AbuseIPDB categories
+      type: string
+  isfetch: false
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python3
```
Comment (yaakovi) [inline Integrations/AbuseDB/AbuseDB.yml]: not sure it is python3, looks more like 2

Comment (orlichter1) [reply Integrations/AbuseDB/AbuseDB.yml]: How do you know?
I usually look for the " Exception as e" and by that I know

Comment (yaakovi) [reply Integrations/AbuseDB/AbuseDB.yml]: `Exception as e` works in both versions.
it looks like an old integration (before we switched to python3

**Comment by orlichter1** [comment]
@kirbles19 Should we generate the .yml again? The difference from the previous version is getting the Indicator fields up to standards.

**Comment by richardbluestone** [comment]
> @kirbles19 Should we generate the .yml again? The difference from the previous version is getting the Indicator fields up to standards.

Hi I've reviewed and updated the .yml file. Please generate the docs.

**Pull request merged**